Dangerfile

```diff
@@ -0,0 +1,23 @@
+# Sometimes it's a README fix, or something like that - which isn't relevant for
+# including in a project's CHANGELOG for example
+declared_trivial = github.pr_title.include? "#trivial"
```

Comment:
how about mention that in our contributing.md ?

Reply:
I think this is a quite minute detail. If you don't know it, nothing bad happens. I think I would not mention it somewhere. Maybe we could add it to the message that Danger will post? Something like: "You can prevent this message by declaring a PR `#trivial` in the title" or such.

Reply:
I added such a message.